staff_features/resources/step_definitions/resource_edit_default_values.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+Given 'the Pre-populate Records option is checked in Repository Preferences' do
+  visit "#{STAFF_URL}/repositories/new"
+
+  fill_in 'repository_repository__repo_code_', with: 'repository_test_resource_edit_default_values'
+  fill_in 'repository_repository__name_', with: 'Repository Test Resource Edit Default Values'
+  find('#repository_repository__publish_').check
+  click_on 'Save'
+
+  message = find('.alert')
+
+  repository_exists = message.text == 'Repository Short Name - The repository short name must be unique within this ArchivesSpace'
+  repository_created = message.text == 'Repository Created'
+  agent_records_message = message.text == 'Agent records cannot be identical'
+
+  expect(repository_exists || repository_created || agent_records_message).to eq true
```

Comment:
It might be a good idea to have a comment that explains what is happening here. This check doesn't have an obvious relationship to the scenario that's being set forth and the multiple || conditions look like the might be a hack of some sort.